hazelcast/src/main/java/com/hazelcast/cluster/impl/JoinMessage.java

```diff
@@ -34,25 +36,25 @@
     protected String uuid;
     protected boolean liteMember;
     protected ConfigCheck configCheck;
-    protected Collection<Address> memberAddresses;
+    protected Collection<Member> members;
```

Comment:
Yes but we wait until 2 sub-clusters become completely disjoint before merging them back. Because of this I still send lite members.